onelogin_rules/onelogin_threshold_accounts_modified.yml

```diff
@@ -0,0 +1,32 @@
+AnalysisType: rule
+Filename: onelogin_threshold_accounts_modified.py
+RuleID: OneLogin.ThresholdAccountsModified
+DisplayName: OneLogin Multiple Accounts Modified
+Enabled: true
+LogTypes:
+  - OneLogin.Events
+Tags:
+  - OneLogin
+Severity: Medium
+Reports:
+  MITRE ATT&CK:
+    - Impact:Account Access Removal
+Description: >
+  Possible Denial of Sevice detected. Threshold for user account password changes exceeded.
```

Comment:
Can you explain the the attack vector here? It seems a single user's password is reset 10 times or more triggers an alert, would it be more useful to look for any 10 users' passwords were reset 10 times?

Reply:
I guess I was thinking that this would trigger off of the `actor_user_name` rather than the `user_name` so if an actor changes 10 different people's passwords, this would trigger. Perhaps I should also add `actor_user_name != user_id` in the rule logic.  This is similar to the `onelogin_password_change.py` rule, but more specifically with a threshold check.  Do you think that's too redundant? Since alerts would fire in that other rule for each event and this one would eventually fire if number of times > threshold?

Reply:
The threshold will be based off of the `dedup` string, in this case:

```
def dedup(event):
    return event.get('user_name')
```

If I'm lining these up correctly. I think if you deduped based off of the `actor_user_name` then you could leave the rest of this rule as is, I see what you're going for now and it makes sense to me.